### ISSUE TITLE
Missing block?

### ISSUE THREAD

**Issue by dankegel** [jsha/blocktogether#291]
I noticed that one account ( BairdMacfarlane ) is blocked by my twitter account @science_shield but missing from @science_shield's blocktogether list.

How does one go about troubleshooting block sync problems?
I can download the blocktogether list, but twitter has disabled blocklist download, so I don't even know how to compare list sizes.

**Comment by dankegel** [comment]
More details:

I used a twitter handle lookup tool to convert BairdMacfarlane to a numerical id, then
searched the downloaded blocklist.  It's present:
```
$ grep 949107908133847041 science_shield-blocklist.csv
949107908133847041
```
And yet "search by screen name" finds no hit for BairdMacfarlane, and his posts are not blocked from my other twitter account which subscribes to the blocklist.  Hrmph.

**Comment by dankegel** [comment]
Also happening with screen name skytroubled

**Comment by jsha** [comment]
I just checked both of these on science_shield's block list, and currently blocktogether.org says "@science_shield blocks X" for both of them. Does this issue still reproduce for you?

**Comment by dankegel** [comment]
Yes.  I'm searching in Actions, and skytroubled still doesn't show up there.
(I do see it in the public search, don't know if it was ever missing there?)
It's also missing in Actions for my personal twitter account that subscribes to @science_shield's blocklist.

**Comment by dankegel** [comment]
Another account in blocklist but not actions:
karipurerock
